pkg/cli-runtime/validation/keyvalue.go

```diff
@@ -63,3 +65,22 @@ func DeletableKeyValues(kvs []string, field string) FieldErrors {
 
 	return errs
 }
+
+func KeyJsonOrYamlValues(kvs []string, field string) FieldErrors {
```

Comment:
nit: change the func name to `JsonOrYamlKeyValues`. reads a bit easy

Reply:
Name changed